accounts-db/src/accounts_db.rs

```diff
@@ -3707,6 +3763,36 @@ impl AccountsDb {
         );
     }
 
+    /// This function handles the case when zero lamport single ref accounts are found during shrink.
+    pub(crate) fn zero_lamport_single_ref_found(&self, slot: Slot, offset: usize) {
+        if let Some(store) = self.storage.get_slot_storage_entry(slot) {
```

Comment:
In `get_slot_storage_entry()`, there is an assert for no shrinking in progress.
No this function is called at startup, clean_unref, and shrink_unref. It is possible that shrink in progress is not empty?
I wonder if we should get rid of the assert in `get_slot_storage_entry`?

Reply:
fixed in https://github.com/anza-xyz/agave/commit/c4d5cda3f77313bc5b5dc7d755f1ba9a96d468a3.